extensions/commands/sbom/cmd_cyclonedx.py

```diff
@@ -0,0 +1,183 @@
+from importlib import import_module
+from functools import partial
+import os.path
+import sys
+from typing import TYPE_CHECKING, Iterable, List, Optional, Set, Tuple, Union
+
+from conan.api.conan_api import ConanAPI
+from conan.api.output import cli_out_write
+from conan.cli.args import common_graph_args, validate_common_graph_args
+from conan.cli.command import conan_command
+from conan.errors import ConanException
+
+if TYPE_CHECKING:
+    from cyclonedx.model.bom import Bom
+
+
+def format_cyclonedx(formatter_module: str, formatter_class: str, bom: 'Bom') -> None:
+    module = import_module(formatter_module)
+    klass = getattr(module, formatter_class)
+    serialized = klass(bom).output_as_string()
+    cli_out_write(serialized)
+
+
+formatter = {
+    "1.4_json": partial(format_cyclonedx, "cyclonedx.output.json", "JsonV1Dot4"),
+    "1.3_json": partial(format_cyclonedx, "cyclonedx.output.json", "JsonV1Dot3"),
+    "1.2_json": partial(format_cyclonedx, "cyclonedx.output.json", "JsonV1Dot2"),
+    "1.4_xml": partial(format_cyclonedx, "cyclonedx.output.xml", "XmlV1Dot4"),
+    "1.3_xml": partial(format_cyclonedx, "cyclonedx.output.xml", "XmlV1Dot3"),
+    "1.2_xml": partial(format_cyclonedx, "cyclonedx.output.xml", "XmlV1Dot2"),
+    "1.1_xml": partial(format_cyclonedx, "cyclonedx.output.xml", "XmlV1Dot1"),
+    "1.0_xml": partial(format_cyclonedx, "cyclonedx.output.xml", "XmlV1Dot0")
+}
+
+
+def format_text(_: 'Bom') -> None:
+    supported = ', '.join([v for v in formatter.keys() if v is not 'text'])
+    raise ConanException(f"Format 'text' not supported. Supported values are: {supported}")
+
+
+formatter["text"] = format_text
+
+
+@conan_command(group="Recipe", formatters=formatter)
```

Comment:
the `group` does not match the path.
shold it not be `SBOM` ?

Reply:
Fixed